src/map-extent.js

```diff
@@ -71,6 +71,45 @@ export class MapExtent extends HTMLElement {
       ? getExtent(this)
       : getCalculatedExtent(this);
   }
+
+  getOuterHTML() {
+    let tempElement = this.cloneNode(true);
```

Comment:
I attempted, but failed to re-write this without cloning the live element.  It's that cloning that yields the console logging due to disconnected map-input and so on (I believe).  That logging seems to interfere with the tests, and what's concerning is that it doesn't appear to happen consistently.